prime-router/src/main/kotlin/Report.kt

```diff
@@ -270,11 +282,7 @@ class Report : Logging {
     val name: String
```

Comment:
It looks like this actually might not be used anymore?  I only saw usages in tests.

Reply:
removed property